scripts/pack.js

```diff
@@ -373,6 +373,13 @@ function getAddonLibs(addonPath) {
     .then((stdout, stderr) => {
       var checks = [];
       for (let line of stdout.trim().split('\n')) {
+        // handle when ldd returns somelib.so => not found as a warning
+        if (!line.startsWith('/')) {
```

Comment:
Is it possible that 'ldd' print a relative path?
I suggest moving the warning into catch block after [ -s ${line} ] like:
exec(`[ -s ${line} ])
  .then(...)
  .catch((e) => {
    // grep the warning and print
  })